tests/test_merging.py

```diff
@@ -2069,7 +2069,7 @@ def test_merge_update_platform(self):
                 # Check a log entry was added for updating the trigraph field
                 results = (
                     self.master_store.session.query(self.master_store.db_classes.Log)
-                    .filter(self.master_store.db_classes.Log.new_value == "PLT")
+                    .filter(self.master_store.db_classes.Log.field == "trigraph")
                     .all()
```

Comment:
shouldn't we also test the value of `previous_value`?

Reply:
We're testing it in database maintenance tests. But, we don't use `previous_value` field while merging databases. So, that's why I changed this query.